app/internal/wm/os_android.go

```diff
@@ -754,6 +765,20 @@ func setCursor(env *C.JNIEnv, view C.jobject, name pointer.CursorName) {
 	callVoidMethod(env, view, gioView.setCursor, jvalue(curID))
 }
 
+func setOrientation(env *C.JNIEnv, view C.jobject, mode OrientationMode) {
+	var id int
+	// Constants defined at https://developer.android.com/reference/android/content/pm/ActivityInfo.
+	switch mode {
+	case UnlockedOrientation:
+		id = 2 // SCREEN_ORIENTATION_USER;
+	case LockedLandscape:
+		id = 0 // SCREEN_ORIENTATION_LANDSCAPE;
```

Comment:
If we're limiting the options to just landscape + portrait for now, let's use the variants that allow free movement, SCREEN_ORIENTATION_USER_LANDSCAPE and SCREEN_ORIENTATION_USER_PORTRAIT.

Reply:
It's only supported by API Level 18. Should it fallback to SCREEN_ORIENTATION_USER if the device is using SDK 16? 😶

Reply:
Yes please, but fall back to SCREEN_ORIENTATION_LANDSCAPE and SCREEN_ORIENTATION_PORTRAIT which is API 1.